resources/scripts/helpers.ts

```diff
@@ -67,3 +67,49 @@ export function hashToPath (hash: string): string {
 export function formatIp (ip: string): string {
     return /([a-f0-9:]+:+)+[a-f0-9]+/.test(ip) ? `[${ip}]` : ip;
 }
+
+/**
+ * Network helpers
+ */
+// Store old values to compare to new values for network speed.
+export let oldBytesReceived = 0;
```

Comment:
These should not be used in this manner, if you were to call these functions in two places at the same time you would be introducing race conditions and get mangled data depending on the order they're called.

The old rates should be tracked in the component itself, and most likely this function doesn't need to be a helper, just a function in the component.

Reply:
I was trying to reduce duplicate code since it would be used in two different locations. I don't know react that well (vue.js for me).

I created the two variables for the chart and the standard text, otherwise ran into where the chart would always be 0.

What way would be good in your view?